src/carina.ts

```diff
@@ -29,6 +31,7 @@ export class Carina extends EventEmitter {
 
     constructor(options: Partial<SocketOptions> = {}) {
         super();
+        this.setMaxListeners(MAX_EVENT_LISTENERS);
```

Comment:
Maybe this should be configurable?

Reply:
I thought about this as well. Let me see how I can do that.